ansible/playbooks/roles/kubernetes_master/templates/kubeadm-config.yml.j2

```diff
@@ -61,3 +61,4 @@ kind: KubeletConfiguration
 apiVersion: kubelet.config.k8s.io/v1beta1
 cgroupDriver: systemd
 rotateCertificates: true
+enableControllerAttachDetach: {{ specification.enable_controller_attach_detach }}
```

Comment:
You have to verify if in any ansible task in kubernetes_master we import specification from kubernetes-node - otherwise variable `enable_controller_attach_detach` could be not accessible.